### PR TITLE
fix: prevent FPs against names due to "axel" and "perl"

### LINKED ISSUE #3490: False positives for 932260 with perl* and axel
### Description

I've run into a handful of false positives on rule 932260 for strings that contain perl and axel, on web forms taking user information. For example, a first name of Axel, a last name of Perlak, email addresses containing either one these, and a city of Perley (Minnesota).

Based on similar FPs for this and other bugs I think this can be fixed by changing the bare axel and perl in to axel@ and perl@, although I'm not sure if there's any secondary commands that this was intended to exclude, such as perldoc. perl5 and perlsh have their own entries, so there may not be anything beyond these.

I'll see if I can put together a pull request.

### How to reproduce the misbehavior (-> curl call)

```
curl -H "x-format-output: txt-matched-rules" https://sandbox.coreruleset.org/events?last_name=Perlak
932260 PL1 Remote Command Execution: Direct Unix Command Execution
949110 PL1 Inbound Anomaly Score Exceeded (Total Score: 5)
980170 PL1 Anomaly Scores: (Inbound Scores: blocking=5, detection=5, per_pl=5-0-0-0, threshold=5) - (Outbound Scores: blocking=0, detection=0, per_pl=0-0-0-0, threshold=4) - (SQLI=0, XSS=0, RFI=0, LFI=0, RCE=5, PHPI=0, HTTP=0, SESS=0, COMBINED_SCORE=5)
```

```
curl -H "x-format-output: txt-matched-rules" https://sandbox.coreruleset.org/events?email=perlak1234@example.com
932260 PL1 Remote Command Execution: Direct Unix Command Execution
949110 PL1 Inbound Anomaly Score Exceeded (Total Score: 5)
980170 PL1 Anomaly Scores: (Inbound Scores: blocking=5, detection=5, per_pl=5-0-0-0, threshold=5) - (Outbound Scores: blocking=0, detection=0, per_pl=0-0-0-0, threshold=4) - (SQLI=0, XSS=0, RFI=0, LFI=0, RCE=5, PHPI=0, HTTP=0, SESS=0, COMBINED_SCORE=5)
```

```
curl -H "x-format-output: txt-matched-rules" https://sandbox.coreruleset.org/events?first_name=Axel 
932260 PL1 Remote Command Execution: Direct Unix Command Execution
949110 PL1 Inbound Anomaly Score Exceeded (Total Score: 5)
980170 PL1 Anomaly Scores: (Inbound Scores: blocking=5, detection=5, per_pl=5-0-0-0, threshold=5) - (Outbound Scores: blocking=0, detection=0, per_pl=0-0-0-0, threshold=4) - (SQLI=0, XSS=0, RFI=0, LFI=0, RCE=5, PHPI=0, HTTP=0, SESS=0, COMBINED_SCORE=5)

```

### Your Environment

* CRS version (e.g., v3.3.4): 4.0.0-rc1
* Paranoia level setting (e.g. PL1) : PL1
* ModSecurity version (e.g., 2.9.6): 2.9.7
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Apache 2.4.57-2
* Operating System and version: Debian 12

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

### PR DESCRIPTION
**Pull request by superlgn**
first attempt at a fix for crs issue #3490 for false positives on 932260 with Perlak, Axel.

Fixes #3490

### CODE REVIEW

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml [hunk 1]
```diff
@@ -380,3 +380,35 @@ tests:
...
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP CRS test agent"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /?last_name=Perlak
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml:395]:
```suggestion
            uri: /get?last_name=Perlak
```

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml [hunk 2]
```diff
@@ -380,3 +380,35 @@ tests:
...
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP CRS test agent"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /?first_name=Axel
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml:411]:
```suggestion
            uri: /get?first_name=Axel
```

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml [hunk 3]
```diff
@@ -380,3 +380,35 @@ tests:
...
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /?last_name=Perlak
+            version: HTTP/1.0
+          output:
+            no_log_contains: id "932260"
+  - test_title: 932260-25
+    desc: "FP with word axel Github Issue #3490"
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml:400]:
```suggestion
    desc: "FP against word 'Axel'"
```

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml [hunk 4]
```diff
@@ -380,3 +380,35 @@ tests:
             version: HTTP/1.0
           output:
             no_log_contains: id "932260"
+  - test_title: 932260-24
+    desc: "FP with word perl Github Issue #3490"
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml:384]: Do not reference GitHub issues please. If we should ever change platforms in the future, that information would become useless.
```suggestion
    desc: "FP against name 'Perlak' (matches 'perl')"
```

**Review by theseion — CHANGES_REQUESTED** [review]
Thanks for your first submission :)

**Review by theseion — DISMISSED** [review]

**Comment by theseion** [comment]
@superlgn the file you touched influences multiple rules. Please run `crs-toolchain regex update -a` and commit the changes. Setup of `crs-toolchain` is documented [here](https://coreruleset.org/docs/development/crs_toolchain/).

**Review by theseion — APPROVED** [review]

**Comment by theseion** [comment]
Thanks!

**Comment by dune73** [comment]
That was a rapid start from reporter to contributor. Thanks @superlgn.

If you share your real name, we'll add you under the real name in the Changelog. Otherwise it's @superlgn.

**Pull request merged**